install/windows-11.md

```diff
@@ -346,7 +345,7 @@ Zum Installieren muss der folgende Befehl ausgeführt werden.
 **Wichtig**: Der Dateiname muss mit dem Namen der heruntergeladenen Datei übereinstimmen.
 Passt den Befehl an, sollte dies nicht der Fall sein.
 ```
-bash Anaconda3-2022.05-Linux-x86_64.sh -p ~/.local/anaconda3 -b
+bash Anaconda3-2023.05-Linux-x86_64.sh -p ~/.local/anaconda3 -b
```

Comment:
Here also the $

Reply:
Will probably lead to the copying of the $ into the terminal.

Reply:
But that happens in general, right?

Reply:
At the start of the linux notes, there is a sentence regarding the $. I would leave Windows as it is and change it with the new installation notes completely.